seedfarmer/commands/_module_commands.py

```diff
@@ -80,6 +81,8 @@ def _env_vars(
         env_vars[_param("PERMISSIONS_BOUNDARY_ARN", use_project_prefix)] = permissions_boundary_arn
     if pypi_mirror_secret is not None:
         env_vars["AWS_CODESEEDER_MIRROR_SECRET"] = pypi_mirror_secret
+    if npm_mirror_secret is not None:
+        env_vars["AWS_CODESEEDER_MIRROR_SECRET"] = npm_mirror_secret
```

Comment:
What if both `pypi_mirror_secret` and `npm_mirror_secret` are set? Is that an invalid config? Right the latter will (silently) stomp over the former.

The tests seem to imply they must be equal?

Reply:
Right now mirror secrets need to be contained within the same Secrets Manager secret e.g.
```
{
    "pypi": {
      ....
    },
    "npm": {
      ....
    }
}
```

We can certainly consider supporting independent secrets though.

Reply:
@clokep good call. +1 for independent secrets actually

Reply:
> Right now mirror secrets need to be contained within the same Secrets Manager secret e.g.

That allows them to be separate in the dictionary. I'm concerned about the env var being passed to code-seeder, which is `AWS_CODESEEDER_MIRROR_SECRET` in both cases.

I'm not very familiar with this code, but as far as I can trace the actual secret string gets stored in this env var (not an encoded JSON object containing both secrets). Is that accurate?

Reply:
My misunderstanding! I think there's a couple options though:

1. Rename various bits so that it is clear that it is shared between PyPI and NPM.
2. Throw an error if they're both provided (and different?)
3. Properly allow them to be different.

Probably any of those would be fine.

Reply:
Yep, good catch we want to support both, so my recent commit addresses your 3rd option.